docs/load.rst

```diff
@@ -327,6 +327,78 @@ from the tables.
     db["emotion"].get()
 
 
+.. _streaming:
+
+Streaming
+---------
+
+:func:`audb.stream` provides a pseudo-streaming mode,
```

Comment:
Pseudo-streaming afaics means: 

there is the local cache to where the data are downloaded to locally(not detailing for now how and when). An iterator works on this local copy and streams chunked data. This differs from huggingfaces datasets that have server-side streaming.

Reply:
The first part is described in the docs:

![image](https://github.com/user-attachments/assets/f421db60-8bf3-4e2b-b02d-8bbf6f9d5562)

So far we have not discussed `huggingface` anywhere in the documentation, so I would not add a statement how this differs compared to `huggingface`.

Reply:
This comment is quite old before I stumbled across `DatabaseIterator.__next__`,  where the imo more relevant discussions were already handled and the conversation closed. So I will resolve this anyway.

I didn't want to prompt a discussion of the design of `huggingface.datasets` here, and this is surely not needed in the documentation. I was just trying to put myself into the shoes of someone coming from that angle.